lib/stateView/document/mongoDbRepository/SVDocumentMongoDbRepository.js

```diff
@@ -17,20 +17,45 @@ class SVDocumentMongoDbRepository {
    * @param {string} documentType
    */
   constructor(mongoClient, convertWhereToMongoDbQuery, validateQuery, documentType) {
-    this.mongoClient = mongoClient.collection(`documents_${documentType}`);
     this.convertWhereToMongoDbQuery = convertWhereToMongoDbQuery;
     this.validateQuery = validateQuery;
     this.documentType = documentType;
+    this.dbName = mongoClient.databaseName;
+    this.collectionName = this.getCollectionName();
+    this.mongoClient = mongoClient.collection(this.collectionName);
+  }
+
+  /**
+   * Returns mongoDatabase collection name
+   *
+   * @private
+   * @returns {string}
+   */
+  getCollectionName() {
+    return `documents_${this.documentType}`;
   }
 
   /**
    * Find SVDocument by id
    *
    * @param {string} id
+   * @param {MongoDBTransaction} [dbTransaction]
    * @returns {Promise<SVDocument>}
    */
-  async find(id) {
-    const result = await this.mongoClient.findOne({ _id: id });
+  async find(id, dbTransaction) {
+    const findQuery = { _id: id };
+
+    let result;
+    if (dbTransaction) {
+      const transactionFunction = async (mongoClient, session) => mongoClient
```

Comment:
Looks not so readable. Maybe add parenthesis around mongoClient call and move it to the next line?